src/sentry/static/sentry/app/views/alerts/rules/details/timeline.tsx

```diff
@@ -247,74 +214,67 @@ type Props = {
 
 class Timeline extends React.Component<Props> {
   renderEmptyMessage = () => {
-    return (
-      <EmptyStateWarning small withIcon={false}>
-        {t('No alerts have been triggered yet')}
-      </EmptyStateWarning>
-    );
+    return <p>{t('No alerts triggered during this time')}</p>;
   };
 
   render() {
     const {api, incidents, orgId, rule} = this.props;
 
     return (
-      <Panel>
-        <PanelHeader>{t('Timeline')}</PanelHeader>
-        <PanelBody>
-          {incidents && rule && incidents.length
-            ? incidents.map(incident => (
-                <TimelineIncident
-                  key={incident.identifier}
-                  api={api}
-                  orgId={orgId}
-                  incident={incident}
-                  rule={rule}
-                />
-              ))
-            : this.renderEmptyMessage()}
-        </PanelBody>
-      </Panel>
+      <History>
+        <SectionHeading>{t('History')}</SectionHeading>
+        <ScrollPanel>
+          <PanelBody withPadding>
+            {incidents && rule && incidents.length
+              ? incidents.map(incident => (
+                  <TimelineIncident
+                    key={incident.identifier}
+                    api={api}
+                    orgId={orgId}
+                    incident={incident}
+                    rule={rule}
+                  />
+                ))
+              : this.renderEmptyMessage()}
+          </PanelBody>
+        </ScrollPanel>
+      </History>
     );
   }
 }
 
 export default Timeline;
 
-const StyledNavTabs = styled(NavTabs)`
-  display: flex;
-  flex-direction: column;
-  margin: ${space(2)};
+const History = styled('div')`
+  margin-bottom: 30px;
 `;
 
-const IncidentHeader = styled('div')`
-  display: flex;
-  margin-bottom: ${space(1.5)};
+const ScrollPanel = styled(Panel)`
+  max-height: 500px;
+  overflow: scroll;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+  &::-webkit-scrollbar {
+    display: none;
+  }
+
+  p {
+    font-size: ${p => p.theme.fontSizeMedium};
+  }
 `;
 
-const AlertBadge = styled('div')<{color: string; icon: React.ReactNode}>`
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  flex-shrink: 0;
-  /* icon warning needs to be treated differently to look visually centered */
-  line-height: ${p => (p.icon === IconWarning ? undefined : 1)};
-  margin-right: ${space(1.5)};
-
-  &:before {
-    content: '';
-    width: 16px;
-    height: 16px;
-    border-radius: ${p => p.theme.borderRadius};
-    background-color: ${p => p.color};
-    transform: rotate(45deg);
+const IncidentSection = styled('div')`
+  &:is(:not(:first-of-type)) {
```

Comment:
```suggestion
  &:not(:first-of-type) {
```
Don't need this pseudo, also not all browsers support it yet